test/raft/README.md

```diff
@@ -0,0 +1,17 @@
+# Running tests with pytest
+
+To run test with pytest execute 
+```bash
+pytest test/raft
+```
+To execute only one file, provide the path filename
+```bash
+pytest test/raft/many_test.cc
+```
+Since it's a normal path, autocompletion works in the terminal out of the box.
```

Comment:
I'm not sure, it doesn't work for me. Autocomplete for `pytest ...` only suggests `.py` files to me. Perhaps pytest hijacks the autocompletions in my setup but it doesn't it yours?

If that's the case then perhaps it's better to remove this line.

Reply:
@piodul I'm guessing you must have somehow configured your shell (which?) to do fancy (but apparently not fancy enough?) autocompletion for pytest. Because without knowing anything about pytest, autocompletion just completes file names - it wouldn't know that ".py" files are any more relevant than ".cc".